Typescript typegen error when using namespaces

Consider the `counter` example from [the crux repo](https://github.com/redbadger/crux/blob/master/examples/counter/shared/src/app.rs).

If I modify it like so:

```diff
diff --git i/examples/counter/shared/src/app.rs w/examples/counter/shared/src/app.rs
index 900e1290..7fb04f7d 100644
--- i/examples/counter/shared/src/app.rs
+++ w/examples/counter/shared/src/app.rs
@@ -26,9 +26,17 @@ pub struct Model {
     count: isize,
 }

+#[derive(Clone, Debug, Serialize, Deserialize, Facet)]
+#[derive(Default)]
+#[facet(namespace = "account")]
+pub struct AccViewModel {
+    pub count: String,
+}
+
 #[derive(Facet, Serialize, Deserialize, Clone, Default)]
 pub struct ViewModel {
     pub count: String,
+    pub acc: AccViewModel,
 }

 #[derive(Default)]
@@ -54,6 +62,7 @@ impl App for Counter {
     fn view(&self, model: &Model) -> ViewModel {
         ViewModel {
             count: format!("Count is: {}", model.count),
+            acc: AccViewModel { count: format!("Count is: {}", model.count) },
         }
     }
 }
```

And then run `just typegen` from the `web-nextjs` folder, I get the following error:

```
app.ts:2:26 - error TS2307: Cannot find module '../account' or its corresponding type declarations.

2 import * as Account from "../account";
                           ~~~~~~~~~~~~


Found 1 error.
```

Which seems to originate from https://github.com/redbadger/facet-generate/blob/301b351defaf345e9251279722ddabb5a4d099d7/crates/facet_generate/src/generation/typescript/emitter/mod.rs#L122

--- a/crates/facet_generate/src/generation/tests/with_namespaces_as_internal/snapshots/typescript/example.ts
+++ b/crates/facet_generate/src/generation/tests/with_namespaces_as_internal/snapshots/typescript/example.ts
@@ -1,4 +1,4 @@
-import * as Other from "../other";
+import * as Other from "./other";
 
 export class Child {
     constructor (public external: Other.OtherParent) {

--- a/crates/facet_generate/src/generation/typescript/emitter/mod.rs
+++ b/crates/facet_generate/src/generation/typescript/emitter/mod.rs
@@ -107,7 +107,7 @@ impl TypeScript {
 impl Module {
     fn ts_namespace_import_path(&self, namespace: &str) -> String {
         self.config().external_packages.get(namespace).map_or_else(
-            || format!("../{namespace}"),
+            || format!("./{namespace}"),
             |path| match &path.location {
                 PackageLocation::Path(_) => {
                     let name = &path.for_namespace;
@@ -135,7 +135,7 @@ impl Emitter<TypeScript> for Module {
             writeln!(w, "{import}")?;
         }
 
-        // Write namespace imports (e.g. `import * as Foo from "../foo";`)
+        // Write namespace imports (e.g. `import * as Foo from "./foo";`)
         let mut import_paths: BTreeMap<String, String> = BTreeMap::new();
         for namespace in referenced_namespaces {
             let import_path = self.ts_namespace_import_path(namespace);

--- a/crates/facet_generate/src/generation/typescript/emitter/tests.rs
+++ b/crates/facet_generate/src/generation/typescript/emitter/tests.rs
@@ -854,7 +854,7 @@ fn type_in_root_and_named_namespace() {
     }
     ");
     insta::assert_snapshot!(root, @r#"
-    import * as Other from "../other";
+    import * as Other from "./other";
     type str = string;
 
     export class Child {

--- a/crates/facet_generate/src/generation/typescript/generator/tests.rs
+++ b/crates/facet_generate/src/generation/typescript/generator/tests.rs
@@ -232,7 +232,7 @@ fn output_adds_import_for_external_namespace() {
     )));
 
     let output = render_output(&config, vec![], &registry);
-    assert!(output.contains(r#"import * as Other from "../other";"#));
+    assert!(output.contains(r#"import * as Other from "./other";"#));
 }
 
 #[test]
@@ -244,7 +244,7 @@ fn output_does_not_import_current_module() {
     )));
 
     let output = render_output(&config, vec![], &registry);
-    assert!(!output.contains(r#"import * as Root from "../root";"#));
+    assert!(!output.contains(r#"import * as Root from "./root";"#));
 }
 
 #[test]
@@ -267,7 +267,7 @@ fn output_uses_external_package_path_for_namespace() {
 
     let output = render_output(&config, vec![], &registry);
     assert!(output.contains(r#"import * as Other from "shared-types";"#));
-    assert!(!output.contains(r#"import * as Other from "../other";"#));
+    assert!(!output.contains(r#"import * as Other from "./other";"#));
 }
 
 #[test]
@@ -340,7 +340,7 @@ fn output_external_package_takes_priority_over_relative_import() {
 
     let output = render_output(&config, vec![], &registry);
     assert!(output.contains(r#"import * as Other from "shared-types";"#));
-    assert!(!output.contains(r#"import * as Other from "../other";"#));
+    assert!(!output.contains(r#"import * as Other from "./other";"#));
 }
 
 #[test]
@@ -352,7 +352,7 @@ fn output_falls_back_to_relative_import_without_external_package() {
     )));
 
     let output = render_output(&config, vec![], &registry);
-    assert!(output.contains(r#"import * as Legacy from "../legacy";"#));
+    assert!(output.contains(r#"import * as Legacy from "./legacy";"#));
 }
 
 #[test]
@@ -500,7 +500,7 @@ fn output_mixed_external_and_local_references() {
 
     let output = render_output(&config, vec![], &registry);
     assert!(output.contains(r#"import * as Other from "shared-types";"#));
-    assert!(!output.contains(r#"import * as Root from "../root";"#));
+    assert!(!output.contains(r#"import * as Root from "./root";"#));
     assert!(output.contains("public local_root: LocalRoot"));
     assert!(output.contains("public local_namespaced: LocalNamespaced"));
     assert!(output.contains("public external: Other.Child"));

